src/osemgrep/language_server/notifications/LoginFinish.ml

```diff
@@ -28,6 +28,7 @@ let of_jsonrpc_params params : (Uri.t * Uuidm.t) option =
 let on_notification server params : unit =
   (* Emulating a poor man's writer's monad, mixed with some LWT goodness. *)
   ignore server;
+  ignore params;
```

Comment:
nit: in the future could we favor prefixing with underscores (e.g. `_server` and `_params`) as opposed to `ignore`? easier to look at whether a parameter is used at the site of the function definition